building_a_catalog_simplified.ipynb

```diff
@@ -0,0 +1,8011 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.          "catalog_name" : "demo_datastore",
</pre><p>In om3-scripts we call this intake_datastore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ACCESS-NRI/access-eval-recipes/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>